File: Day 104/homework/hw4.py
# 4) მომხმარებელს შემოატანინეთ ორი რიცხვი და სცადეთ მათი გაყოფა.
# თუ მეორე რიცხვი არის 0, დაიჭირეთ შეცდომა და დაბეჭდეთ:
# “Can’t divide a number by 0.”


num1 = int(input("Enter your first number: "))
num2 = int(input("Enter your second number: "))

try:
    num1 // num2
except:
    print("Can’t divide a number by 0")

File: Day 133/homework/hw4.js
// 4) შექმენით ობიექტი calculator, რომელსაც ექნება ორი property: num1 და num2. დაამატეთ ამ ობიექტს მეთოდი add, რომელიც დააბრუნებს num1-ისა და num2-ის ჯამს. გამოიძახეთ ეს მეთოდი და დაბეჭდეთ შედეგი


const calculator  = {
    num1: 67,
    num2: 19,

    add() {
        return this.num1  + this.num2
    }
}


console.log(calculator.add())